crates/router/src/connector/stripe/transformers.rs

```diff
@@ -290,9 +290,9 @@ pub struct StripeSofort {
     #[serde(rename = "payment_method_data[type]")]
     pub payment_method_data_type: StripePaymentMethodType,
     #[serde(rename = "payment_method_options[sofort][preferred_language]")]
-    preferred_language: String,
+    preferred_language: Option<String>,
     #[serde(rename = "payment_method_data[sofort][country]")]
-    country: api_enums::CountryAlpha2,
+    country: Option<api_enums::CountryAlpha2>,
```

Comment:
is this not mandatory??

Reply:
No country is not mandatory for [stripe](https://stripe.com/docs/payments/sofort/accept-a-payment?platform=web&ui=API#create-payment-intent)

Reply:
as we checked , it is mandatory. make the necessary changes